templates/operator/templates/stateful-set.yaml.template

```diff
@@ -148,6 +154,17 @@ spec:
             - name: node-readiness-probe-sh
               mountPath: /etc/cassandra/node-readiness-probe.sh
               subPath: node-readiness-probe.sh
+            - name: cassandra-home
+              mountPath: /home/cassandra/.cassandra/
```

Comment:
Should `cassandra-home` be just `/home/cassandra/`?

Reply:
Both cqlsh and nodetool creates files in `.cassandra`, /home/cassandra is not used directly
so directly mounted it there, otherwise it would need another mkdir.

> cassandra@cassandra-node-0:~ $ ls -lah
total 0
drwxr-xr-x. 3 root root      24 Jan 29 14:46 .
drwxr-xr-x. 1 root root      23 Jan 29 14:46 ..
drwxrwsrwx. 2 root cassandra 66 Jan 29 14:49 .cassandra
cassandra@cassandra-node-0:~$ ls -lah .cassandra/
total 12K
drwxrwsrwx. 2 root      cassandra   66 Jan 29 14:49 .
drwxr-xr-x. 3 root      root        24 Jan 29 14:46 ..
-rw-------. 1 cassandra cassandra    5 Jan 29 14:49 cqlsh_history
-rw-r--r--. 1 cassandra cassandra  264 Jan 29 14:46 cqlshrc
-rw-r--r--. 1 cassandra cassandra 1.7K Jan 29 14:50 nodetool.history

Reply:
Let's name it something other than "cassandra-home" then, since it implies `/home/cassandra`. Maybe something like `cassandra-configuration-directory` or `dot-cassandra`.

Reply:
`dot-cassandra` SGTM :+1: